NMS-15027: Add JSON support to the Kafka producer for flows (#5497)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/flows/setup.adoc

Enriched flows (with OpenNMS node data) can also be forwarded to Kafka.

> Enriched flows are stored in `flowDocuments` topic and the payloads are encoded using link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers (GPB)] by default.

Reviewer: ```suggestion
Enriched flows are stored in `flowDocuments` topic and the payloads are encoded using link:https://developers.google.com/protocol-buffers/[Google Protocol Buffers] (GPB) by default.
```

See `flowdocument.proto` in the corresponding source distribution for the model definitions.

> JSON support can be enabled by setting the `useJson` property to `true`.

Reviewer: ```suggestion
To enable JSON support, set the `useJson` property to `true`.
```
